build/release.sh

```diff
@@ -7,8 +7,8 @@ function display_help ()
 {
     echo "\
 usage: release.sh [--output-directory=<binary_output_directory>]
-                  (--platform=<argon|argon-som|boron|boron-som...
-                  |core|electron|p1|photon|xenon|xenon-som>...
+                  (--platform=<argon|asom|boron|bsom...
```

Comment:
I realize this isn't the focus of this PR but it seems there is room for refactoring in these bash scripts to remove the duplication. cc: @zfields